quality-assurance/test-strategy.md

```diff
@@ -84,9 +95,11 @@ The disaster recovery plan should be tested for a new or significantly changed s
 
 ### Functional
 
-![Test Pyramid](images/test-pyramid.png)
+![Test Pyramid](images/the_test_pyramid.png)
 
+The test pyramid indicates where the test effort should be focused in order to provide a better return and a faster feedback loop for failures and defects.
 Tests should be added at the correct level of the test pyramid – lower is better.
+The Functional Testing will be focused on the solutions functionality and business logic validation process. 
```

Comment:
Move this sentence to the top of the section.  Then needs a sentence leading into the idea that functional testing can happen at a number of levels, and that these have different ROIs.
solutions -> solution's
remove "validation process"?